flink/src/main/java/org/apache/iceberg/flink/sink/FlinkSink.java

```diff
@@ -135,6 +135,13 @@ private Builder forRowData(DataStream<RowData> newRowDataInput) {
       return this;
     }
 
+    private <T> Builder forMapperOutputType(DataStream<T> input,
+                                        MapFunction<T, RowData> mapper,
```

Comment:
Nit: missing a code format.

Reply:
thx. will fix (along with any other changes if needed)

Reply:
Yeah,  this is needed.

Reply:
fixed